osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -164,27 +184,31 @@ private void transformPopOut(int newValue)
         {
             ((IHasText)popOutCount).Text = formatCount(newValue);
 
-            popOutCount.ScaleTo(1.6f);
-            popOutCount.FadeTo(0.75f);
-            popOutCount.MoveTo(Vector2.Zero);
+            popOutCount.ScaleTo(1.56f);
+            popOutCount.FadeTo(0.6f);
 
-            popOutCount.ScaleTo(1, pop_out_duration);
-            popOutCount.FadeOut(pop_out_duration);
-            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration);
+            popOutCount.ScaleTo(1, big_pop_out_duration);
+            popOutCount.FadeOut(big_pop_out_duration);
         }
 
         private void transformNoPopOut(int newValue)
         {
             ((IHasText)displayedCountSpriteText).Text = formatCount(newValue);
 
+            counterContainer.Size = displayedCountSpriteText.Size;
+
             displayedCountSpriteText.ScaleTo(1);
         }
 
         private void transformPopOutSmall(int newValue)
         {
             ((IHasText)displayedCountSpriteText).Text = formatCount(newValue);
-            displayedCountSpriteText.ScaleTo(1.1f);
-            displayedCountSpriteText.ScaleTo(1, pop_out_duration);
+
+            counterContainer.Size = displayedCountSpriteText.Size;
+
+            displayedCountSpriteText.ScaleTo(1).Then()
```

Comment:
I intentionally made these changes during implementation to make the animations feel better. I guess if we're going for 1:1 compatibility then this is fine.